app/src/main/java/com/nervousfish/nervousfish/activities/QRExchangeKeyActivity.java

```diff
@@ -99,22 +95,21 @@ private void drawQRCode() {
      * {@inheritDoc}
      */
     @Override
-    public void onActivityResult(final int requestCode, final int resultCode, final Intent intent) {
+    public void onActivityResult(final int requestCode, final int resultCode, final Intent data) {
```

Comment:
I think a prefer the third argument to be named `intent` (instead of `data`) because an `Intent` is not the same as "raw" data 🤔